src/main/java/edu/kis/powp/jobs2d/commands/OperateToCommand.java

```diff
@@ -0,0 +1,17 @@
+package edu.kis.powp.jobs2d.commands;
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public class OperateToCommand implements DriverCommand {
+    private int x;
+    private int y;
+
+    public OperateToCommand(int x, int y){
+        this.x = x;
+        this.y = y;
+    }
+
+    @Override
+    public void execute(Job2dDriver driver) {
+		driver.operateTo(x,y);
+	}
```

Comment:
Original code used tabs instead of spaces. It is important to keep the original convention. It makes tracking actual changes more difficult.